python/tk_multi_publish2/dialog.py

```diff
@@ -235,8 +234,26 @@ def __init__(self, parent=None):
         self._display_action_name = self._bundle.get_setting("display_action_name")
         self.ui.publish.setText(self._display_action_name)
 
+        # UI tweaks based on the 'manual_load_enabled' property
```

Comment:
do we need to hide the drop area now that we're going to the other widget in the stack? it's not possible to get back to this page as far as i know.

Reply:
Good point! The actual functionality is disable regardless of UI.
Done.